sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/ColumnStats.scala

```diff
@@ -296,14 +296,11 @@ private[columnar] final class BinaryColumnStats extends ColumnStats {
 }
 
 private[columnar] final class IntervalColumnStats extends ColumnStats {
-  protected var upper: CalendarInterval = CalendarInterval.MIN_VALUE
-  protected var lower: CalendarInterval = CalendarInterval.MAX_VALUE
+  protected var upper: CalendarInterval = null
+  protected var lower: CalendarInterval = null
```

Comment:
can we follow `BinaryColumnStats` and don't define the `upper` and `lower` variable?

Reply:
yes